functions/zoom-downloader.py

```diff
@@ -131,33 +132,33 @@ def host_name(self):
                                 )
         return self._host_name
 
+    @property
+    def uuid(self):
+        return self.data["uuid"]
+
+    @property
+    def zoom_series_id(self):
+        return self.data["zoom_series_id"]
+
     @property
     def recording_files(self):
         if not hasattr(self, "_recording_files"):
+
             files = self.data["recording_files"]
-            start_times = sorted(
-                set([file["recording_start"] for file in files])
-            )
-            track_numbers = {
-                start_times[i]: i for i in range(len(start_times))
-            }
+            start_times = sorted(set(
+                [ file["recording_start"] for file in files ]
+            ))
 
-            tracks = [{}] * len(start_times)
+            zoom_files = []
             for file in files:
-                track_number = track_numbers[file["recording_start"]]
-                tracks[track_number][file["recording_type"]] = file
-
-            logger.info({"tracks": tracks})
-
-            self._recording_files = []
-            for track_sequence in range(len(tracks)):
-                for file_data in tracks[track_sequence].values():
-                    file_data["meeting_uuid"] = self.data["uuid"]
-                    file_data["zoom_series_id"] = self.data["zoom_series_id"]
-                    file_data["created_local"] = self._created_local
-                    self._recording_files.append(
-                        ZoomFile(file_data, track_sequence)
-                    )
+                track_seq = start_times.index(file["recording_start"])
+                file["meeting_uuid"] = self.uuid
+                file["zoom_series_id"] = self.zoom_series_id
+                file["created_local"] = self._created_local
+                zoom_files.append(ZoomFile(file, track_seq))
+
+            self._recording_files = zoom_files
+
```

Comment:
Would be nice to have:
logger.info(self._recording_files)
here

Reply:
it's just going to print a bunch of `<ZoomFile...>` items, no?. You could make a separate PR to define a `__repr__` method for the ZoomFile class, and then add the log event.